src/domains/chart/utils/get-portal-node-styles.ts

```diff
@@ -37,7 +37,10 @@ export const getPortalNodeStyles: GetPortalNodeStyles = (
     : null
 
   if (heightFromLocalStorage) {
-    height = heightFromLocalStorage
+    // .replace() is for backwards compatibility -  old dashboard was always doing
+    // JSON.stringify when setting localStorage so many users have '"180px"' values set.
+    // We can remove .replace() after some time
+    height = heightFromLocalStorage.replace(/"/g, "")
```

Comment:
Could't be there something like `"\"some\""`?

Reply:
@builat no, it's just for this specific property type. if you go to any demo netdata page and do `localStorage["chart_heights.frankfurt_netdata_rocks_system_cpu"]`, you'll get `""180px""`

Reply:
it's just for reading properties saved by old dashboard. new dashboard will just save normal strings, without quotes